debian/rules

```diff
@@ -59,6 +59,7 @@ override_dh_installinit:
 	install -d -m0755 debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-osd*.conf debian/ceph-osd/etc/init
 	install -m0644 src/upstart/ceph-disk.conf debian/ceph-osd/etc/init
+	install -m0644 build/src/upstart/ceph-osd.conf debian/ceph-osd/etc/init
```

Comment:
on a second thought, i think we should leave the upstart scripts as they are, and remove them in a separated commit, as upstart is not used in any of supported distros [1]. @ktdreyer @liewegas, what do you think?

---
[1] https://github.com/ceph/ceph/blob/master/doc/start/os-recommendations.rst#infernalis-92z-and-jewel-102z